### ISSUE TITLE
Tiered Discount - Split discount evenly between items

### ISSUE THREAD

**Issue by verniesanders** [jgodson/shopify-script-creator#95]
Thank you for creating this script creator, it is a lifesaver!

I am trying to create the following tiered discount when the code "PRESDAY" is entered at checkout:

Spend $150, get $25 off
Spend $250, get $50 off
Spend $300, get $75 off

I've selected the "Tiered Discount" campaign, added the Cart Qualifier "Cart has discount code" and entered in my code, then selected "fixed total discount" under discount type and "cart subtotal" under tier type.

The script generated worked great, but it applies the discount to each line item instead of the entire order. I noticed in the "conditional discount" campaign type there is an option to split the discount between items, and I'm wondering if there is a way to implement that logic for the tiered campaign? 

Basically, if I have a cart subtotal of $150 with two items, instead of getting $25 off each item, I would get $25 off the total cart value. 

I hope this makes sense! I attached photos of the setup in the script editor you created, as well as the preview of the script in the Shopify script editor. Any help is greatly appreciated!

<img width="385" alt="Screenshot 2024-02-07 at 8 51 29 PM" src="https://github.com/jgodson/shopify-script-creator/assets/159407095/e091a06a-2a54-4284-8995-cf30242dab09">

<img width="308" alt="Screenshot 2024-02-07 at 8 52 02 PM" src="https://github.com/jgodson/shopify-script-creator/assets/159407095/a51ea85c-c8d3-4db2-82da-eb720542fa8d">

**Comment by verniesanders** [comment]
I am so silly, after posting this, I realized the script was already functioning as intended. Thank you again!
